Client/Frontend/Browser/TabManager.swift

```diff
@@ -229,6 +229,14 @@ class TabManager : NSObject {
         let storedTabs: [RemoteTab] = optFilter(tabs.map(Browser.toTab))
         storage?.insertOrUpdateTabs(storedTabs)
     }
+
+    func didReceiveMemoryWarning() {
+        for (index, tab) in enumerate(tabs) {
+            if selectedIndex != index {
+                tab.zombify()
```

Comment:
I still think we should avoid doing our own zombification, for a few reasons:
 1. The web view already does it for us. Why reinvent the wheel and potentially introduce new bugs for existing functionality?
 1. As I mentioned before, the OS zombifies each web view on an as-needed basis, rather than all at once. If I have 20 tabs open and I get a low memory warning, it's a much better user experience if only a couple of those tabs are closed as opposed to all of them.